packages/lesswrong/lib/filterSettings.ts

```diff
@@ -22,6 +23,15 @@ export const FILTER_MODE_CHOICES = [
 ] as const;
 export type FilterMode = typeof FILTER_MODE_CHOICES[number]|"TagDefault"|number
 
+const STANDARD_FILTER_MODES: (string|number)[] = [...FILTER_MODE_CHOICES, 0, 0.5, 25];
+const OLD_FILTER_MODES: (string|number)[] = [-25, -10, 10];
```

Comment:
Which you might have done, but not tried 0.5. Or it might not trigger a bug maybe.

Reply:
We could include `STANDARD_FILTER_MODES` here, but the way this is used in `isCustomFilterMode` it would be redundant as we check `STANDARD_FILTER_MODES` afterwards anyway. There's definitely an argument to be made that if these values are tailored to this function then they should only be declared inside the function - it's potentially worse for perfomance if the JS engine isn't smart enough to detect that these values are static but, realistically, it shouldn't matter here.

I did test on LW, but it looks to me that not including 0.5 there is currently a bug as there's an "0.5x" button:
<img width="560" alt="Screenshot 2022-07-06 at 10 46 15" src="https://user-images.githubusercontent.com/5075628/177522636-ecaeb2c1-7bbd-4b3e-add1-a7e0b58e9b22.png">
I could be wrong though?

Reply:
I am a lying liar. And also failed at reading. Nevermind!